src/main/java/reactor/netty/transport/ProxyProvider.java

```diff
@@ -389,6 +406,13 @@ public ProxyProvider build() {
 		 */
 		Builder httpHeaders(Consumer<HttpHeaders> headers);
 
+		/**
+		 * The proxy connect timeout in millis.
+		 * @param connectTimeoutMillis
+		 * @return
+		 */
+		Builder connectTimeoutMillis(long connectTimeoutMillis);
```

Comment:
Please provide javadoc explaining the default value, what happens when it is a non positive value.